docs/wave.rst

```diff
@@ -42,7 +43,7 @@ receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`da
   data.set_receiver_pos(receiver_coords)
   data.set_shape(nt, 101)
 
```

Comment:
How to explain spacing, origin and dimensions?. I thought they''re just geophysics data or default data. What I know is that they contribute to getting the shape of grids, calculating dt, defining the source coordinate as well as the velocity model. But I don't know why they're set in this way like why the dimensions of origin and spacing are both two?